README.md

```diff
@@ -38,14 +38,17 @@ New commands should go under commands.
 Ensure you are running go 1.7 or later.
 
 Install glide first [it's on GitHub](https://github.com/Masterminds/glide)
+```
+go get github.com/Masterminds/glide
+```
 
 ```
     go get github.com/feedhenry/rhm
     cd $GOPATH/src/github.com/feedhenry/rhm
     glide install
-    go build .
+    go install
```

Comment:
I just tested this locally, and if I remove the `rhm` from my binary and run `glide install` then the `rhm` binary is not recreated, so I think this is required?